ci/prow/make_config.go

```diff
@@ -1330,7 +1330,7 @@ func generateTestGroup(repoName string, jobNames []string) {
 		gcsLogDir := fmt.Sprintf("%s/%s/%s", gcsBucket, logsDir, testGroupName)
 		extras := make(map[string]string)
 		switch jobName {
-		case "continuous", "dot-release", "auto-release", "performance", "latency", "playground", "nightly":
+		case "continuous", "dot-release", "auto-release", "performance", "latency", "playground", "nightly", "istio-1.0.7-mesh", "istio-1.0.7-no-mesh", "istio-1.1.2-mesh", "istio-1.1.2-no-mesh":
```

Comment:
Can we have a more generic approach here for custom jobs, instead of hardcoding too-specific job names?

Reply:
I doubt it. Custom jobs allows full control over what to run, we would effectively be defining what custom job is if we do that.

A custom type, say e2e_tests would be better suited as then we could guarantee that it is running the e2e_tests.sh script, which we know how to configure.

Reply:
Generalize the config generation for custom jobs, don't mind the naming. Because that's effectively what you are doing here.

Reply:
I'm confused now.

what is the change you want me to make?

Reply:
Nevermind. The change needs to be deeper than that, as the generated tab names had bad readability. I'll file an issue for that, let's get this coverage in.